platform/integrations/gitlab-ci.mdx

```diff
@@ -55,48 +66,4 @@ e2e:
    corresponding namespace for the virtual cluster is deleted. By using `after_script` we can ensure the virtual cluster
    is deleted even if the tests fail. You may wish to skip this step if reusing the virtual cluster with the `--upgrade` flag when creating the virtual cluster.
 
-## Spaces for Merge Requests
```

Comment:
Why did we delete this?

Spaces was renamed to Namespaces.

Reply:
The instructions were moved to a tip in the first section since its no longer subcommand of something to create.  Instead, the `--namespace` flag will create the `space` since the `--create-namespace` flag defaults to true.